samples/SocketsSample/project.json

```diff
@@ -55,7 +55,7 @@
   },
 
   "scripts": {
-    "precompile": [ "tsc --project ../../src/Microsoft.AspNetCore.SignalR.Client.TS/ --out wwwroot/js/signalr-client.js" ],
+    "precompile": [ "../../bin/nodejs/node_modules/.bin/tsc --project ../../src/Microsoft.AspNetCore.SignalR.Client.TS/ --out wwwroot/js/signalr-client.js" ],
```

Comment:
Only it may not work - see: https://github.com/Microsoft/TypeScript/issues/5995#issuecomment-254408809

As you said - let's get this in to unblock the CI and then try fix VS.